p2p/src/authenticated/actors/tracker/actor.rs

```diff
@@ -467,10 +467,10 @@ impl<E: Spawner + Rng + Clock + GClock + Metrics, C: Scheme> Actor<E, C> {
         // Parse bit vector bytes
         let bits: BitVec<u8, Lsb0> = BitVec::from_vec(bit_vec.bits);
```

Comment:
I'm all for removing this external dependency and just doing our own BitVec that has an API for this math FWIW